hasher-matcher-actioner/hmalib/lambdas/api/content.py

```diff
@@ -16,7 +16,7 @@
 
 from hmalib.lambdas.api.middleware import jsoninator, JSONifiable, DictParseable
 from hmalib.models import MatchRecord, PipelineHashRecord
-from hmalib.common.content_models import (
```

Comment:
I think if the db models also works as a message that is maybe okay (but not vice versa, i.e. models/ is the source of truth), I think there is value either way it terms of rules here, being strictly isolated is only worth it if we can enforce programmatic imo. Since that is not trivial addition to the readme/docstrings about content_object being a little special is good enough for me.